lms/templates/problem_notifications.html

```diff
@@ -4,9 +4,10 @@
 from openedx.core.djangolib.markup import HTML
 %>
 
-<div class="notification ${notification_type}" tabindex="-1">
+<div class="notification ${notification_type} ${'notification-hint' if hint_notification else 'notification-response'}" tabindex="-1">
```

Comment:
I see-- so there can only ever be a single notification-response visible, and we are using the classname to get it for focusing. That seems quite brittle, as there might be multiple "notification-response" shown at once in the future. In particular, what about Save? Can the Save notification and a response notification show at the same time? FYI @staubina 

What I would prefer (though I do not know if this is feasible) is to get a reference to the specific notification that was created, and use that for focusing. My fear is that this is not possible though because the creation of the notification happens in the problem render, and we don't know which (if any) notifications were actually created.